searxng/searxng
# [mod] Update input when selecting autocomplete prediction with keyboard

## What does this PR do?

- Update input when selecting autocomplete prediction with keyboard
- Search immediately by pressing enter key
- Search immediately by clicking on an autocomplete suggestion

## Why is this change important?

- https://github.com/searxng/searxng/issues/778


## How to test this PR locally?

1. run the project with this branch
1. start typing request in search input with autocomplete enabled
1. use arrow (up/down) keys to select suggestions
1. notice that an active autocomplete suggestion appears in search input
1. press enter key or click on a suggestion to immediately submit your search request


## Author's checklist

## Related issues

Closes: https://github.com/searxng/searxng/issues/778
